man/systemd-cryptenroll.xml

```diff
@@ -256,6 +256,14 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--empty</option></term>
+
+        <listitem><para>Enroll an empty key. This has the effect of disabling encryption on the volume
+        without decrypting it. Unlike an empty <option>--password</option>, a key enrolled via this
```

Comment:
@keszybz is right, this is a bit confusing, i.e. encryption is not disabled. What this does is that you can unlock the encryption without providing any key.

So maybe "This has the effect of allowing unlocking of the volume without providing a key, thus disabling confidentiality protections. The volume remains encrypted however, and thus, if at a later point the empty key enrollment is removed again protections are reenabled." 

or something like that.